cmd/docs.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/TouchBistro/goutils/fatal"
+	"github.com/TouchBistro/tb/config"
+	"github.com/spf13/cobra"
+)
+
+var docsCmd = &cobra.Command{
+	Use:   "docs",
+	Args:  cobra.MinimumNArgs(1),
+	Short: "Opens link to API docs for a given service",
+	Run: func(cmd *cobra.Command, args []string) {
+		serviceName := args[0]
+		service, err := config.LoadedServices().Get(serviceName)
+		if err != nil {
+			fatal.ExitErrf(err, "%s is not a valid service.\nTry running `tb list` to see available services\n", serviceName)
+		}
+
+		if !service.HasGitRepo() {
+			fatal.Exitf("%s does not have a repo or is a third-party repo\n", serviceName)
+		}
+
+		docsURL := service.EnvVars["API_DOCS_URL"]
+		if docsURL == "" {
+			fatal.Exitf("API_DOCS_URL environment variable not found for service %s\n", serviceName)
+		}
+
+		fmt.Printf("Opening docs for %s...\n", serviceName)
+		openDocs(docsURL)
+	},
+}
+
+func openDocs(url string) {
+	cmd := exec.Command("open", url)
```

Comment:
`open` is macOS only and tb is supported on linux now so this needs to be guarded with an OS check and have a way for it to work on linux as well. I have some helpers in `util` for checking the OS